resources/application-connector/values.yaml

```diff
@@ -42,31 +42,31 @@ global:
       version: "PR-13670"
     application_connectivity_certs_setup_job:
       name: "application-connectivity-certs-setup-job"
-      version: "PR-13170"
+      version: "708f6a87"
     application_connectivity_validator:
       name: "application-connectivity-validator"
-      version: "PR-13170"
+      version: "cc89c542"
     application_gateway:
       name: "application-gateway"
-      version: "PR-13171"
+      version: "6d430445"
     application_operator:
       name: "application-operator"
-      version: "PR-12257"
+      version: "3f163e8f"
     application_registry:
       name: "application-registry"
-      version: "PR-12257"
+      version: "a0a4ec40"
     central_application_connectivity_validator:
       name: "central-application-connectivity-validator"
-      version: "PR-12963"
+      version: "708f6a87"
     central_application_gateway:
       name: "central-application-gateway"
-      version: "PR-13171"
+      version: "708f6a87"
```

Comment:
PR `PR-13171` is merged as a commit [6d43044](https://github.com/kyma-project/kyma/commit/6d430445a34a50e11d246ff3e0195019ad2598f8). Therefore, the bumped image should be `6d430445`